SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -54,6 +55,7 @@ void UGlobalStateManager::Init(USpatialNetDriver* InNetDriver)
   
 	bAcceptingPlayers = false;
 	bCanBeginPlay = false;
+	bCanSpawnWithAuthority = false;
```

Comment:
```suggestion
	bAssumeAuthorityOnStartUpActors = false;
```

Reply:
Or maybe not, bit nitpicky but I got confused thinking that the bool controlled authority on the (I know, single) GSM